File: 151001/Novik/RESTLab1/REST/Mapper/LabelMapper.cs
using AutoMapper;
using REST.Models.DTOs.RequestIn;
using REST.Models.DTOs.Responses;
using REST.Models.Entity;

namespace REST.Mapper;

public class LabelMapper:Profile
{
    public LabelMapper()
    {
        CreateMap<Label, LabelResponseTo>();
        CreateMap<LabelRequestTo, Label>();
    }
}

File: 151002/Samusenko/lab_five/publisher/src/main/java/com/example/publisher/service/mapper/LabelMapper.java
package com.example.publisher.service.mapper;
import com.example.publisher.model.entity.Label;
import com.example.publisher.model.request.LabelRequestTo;
import com.example.publisher.model.response.LabelResponseTo;
import org.mapstruct.Mapper;

import java.util.List;

@Mapper(componentModel = "spring")
public interface LabelMapper {
    LabelResponseTo getResponse(Label label);
    List<LabelResponseTo> getListResponse(Iterable<Label> labels);
    Label getLabel(LabelRequestTo labelRequestTo);
}
